trieste/acquisition/rule.py

```diff
@@ -54,12 +56,12 @@ class AcquisitionRule(ABC, Generic[S, SP_contra]):
     """ The central component of the acquisition API. """
 
     @abstractmethod
-    def acquire(
+    def acquire_multi(
```

Comment:
if this is the main function, it should be called `acquire`? and we use `acquire_single` (or a better name) for the single model?

Reply:
Could do, though since people are more likely to be using the single model version I thought it made more sense this way?

The only reason there are two versions is to avoid forcing people who write new Acqusition Rules to explicitly deal with the 
single model case each time. This way, they write the more complex `acquire_multi` for their rule, and get the simpler `acquire` for free.

Reply:
Not sure what people would use most, multi or single model case. @henrymoss any opinion on what should be the default?

Reply:
Do some rules only work foor single models?

Reply:
The ThompsonSampling rule currently assumes a single model.

Reply:
Perhaps @vpicheny means that we should still request people write the more complex multi-model acquire as the default, but just name that acquire. With the single-model one coming for free, but named acquire_single?

Reply:
yes that's what I meant.

Reply:
Following a chat, Henry convinced me that simplifying the naming for people writing new rules is more important than simplifying it for users running single model experiments, as the latter won't typically be exposed to the acquire method anyway since it's called automatically the optimization loop. I've changed it as suggested.